Sources/Data/OAMapLayersConfiguration.h

```diff
@@ -31,13 +31,13 @@
 
 @interface OAMapLayersConfiguration : NSObject <NSCoding>
 
-- (instancetype) initWithHiddenLayers:(NSMutableSet *)hiddenLayers;
+- (instancetype) initWithHiddenLayers:(NSMutableArray *)hiddenLayers;
```

Comment:
This is incorrect. You should not change the implementation of the class. Instead, you should create an array from set in the profile setting.